lib/algolia/analytics.rb

```diff
@@ -0,0 +1,77 @@
+module Algolia
+
+  class Analytics
+    attr_reader :ssl, :ssl_version, :headers
+
+    def initialize(client, params)
+      @client = client
+
+      @headers      = params[:headers]
+      @ssl          = params[:ssl].nil? ? true : params[:ssl]
+      @ssl_version  = params[:ssl_version].nil? ? nil : params[:ssl_version]
```

Comment:
I'm not sure to understand what's the intent here? In any case, it is equal to `params[:ssl_version]`. So this should be enough: `@ssl_version = params[:ssl_version]`

Reply:
If you use `init_analytics` yes, but if you build it yourself, you might not pass it. Also, at the the time I wrote this, I didn't pass the client to analytics. Maybe I could just read these value from the client 🤔

Reply:
Yeah, if you don;t pass it `@ssl_version = params[:ssl_version]` then `@ssl_version` will be `nil`, exactly like `params[:ssl_version].nil? ? nil`